### PR TITLE
summer 2021

### PR DESCRIPTION
**Pull request by vivian-dai**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -122,13 +122,29 @@ <h1>
           <div
             class="swiper-slide work"
             data-hash="work-1"
+            data-color="#90919c"
```
Comment (shari09) [inline index.html:125]: i hope this isnt some ugly bright colour

Comment (vivian-dai) [reply index.html:125]: this is grey 😭

**Review by shari09 — COMMENTED** [review]
im not the reviewer so i'll just comment without approval

**Review by wang-joseph — APPROVED** [review]
lgtm!!!!

**Pull request merged**